2024.html

```diff
@@ -69,6 +69,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
             <li><a href="https://knowler.dev/">Nathan Knowler</a></li>
             <li><a href="https://robin.is/">Robin Massart</a></li>
             <li>Jens Oliver Meiert: <a href="https://frontenddogma.com/">Frontend Dogma</a></li>
+            <li><a href="https://lostfocus.de/">Dominik Schwind's LostFocus</a></li>
```

Comment:
Minor (we might change this later, however): What do you think of the format as in the line above, indicating and sorting by your name, then linking “LostFocus”?

Reply:
Okay!